app/views/layouts/application.html.erb

```diff
@@ -7,11 +7,16 @@
 
     <%= stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload' %>
     <%= javascript_pack_tag 'application', 'data-turbolinks-track': 'reload' %>
+    <%= javascript_pack_tag 'scroll' %>
   </head>
 
   <body>
-    <p class="notice"><%= notice %></p>
-    <p class="alert"><%= alert %></p>
-    <%= yield %>
+    <%= render 'shared/header' %>
+    <div class="start-block">
+      <p class="notice"><%= notice %></p>
```

Comment:
if notice and alert is empty? we will have 2 empty "p" on the page

Reply:
fixed